internal/policies/manager.go

```diff
@@ -1,3 +1,25 @@
+// Package policies - Policy manager guidelines
+//
+// ADSys is expected to apply configuration policies, not enforce them. We are responsible for
+// properly configuring those policies and set up the machine to apply them, but we can not ensure that
+// they will be executed as this relies on lots of variables and system functionalities.
+// As such, policy managers are expected to only prevent authentication when something goes wrong
+// during the configuration of the policies, whilst ensuring that the user will be warned should a
+// configuration fail to execute.
+//
+// We should prevent authentication on errors such as:
+//   - failed to copy the policy requested assets to the machine;
+//   - failed to parse policy configuration values;
+//   - failed to write the requested files on disk;
```

Comment:
We can add a 4th case: missing binary or binary execution to enforce a policy not present (dconf update, apparmor_parser not present…)